module/data/item/templates/activities.mjs

```diff
@@ -167,46 +167,60 @@ export default class ActivitiesTemplate extends SystemDataModel {
    */
   static #migrateUses(source) {
     // Remove any old ternary operators from uses to prevent errors
-    if ( source.uses?.max?.includes(" ? ") ) source.uses.max = "";
+    let maxIncludesIsAFunction = source?.uses && typeof (source?.uses?.max?.includes) === "function";
+
     for ( const activity of Object.values(source.activities ?? {}) ) {
-      if ( activity?.uses?.max?.includes(" ? ") ) activity.uses.max = "";
+      if (maxIncludesIsAFunction && activity?.uses?.max?.includes(" ? ") )
+      {
+        activity.uses.max = "";
+      }
```

Comment:
We should just be able to optionally chain on the function itself (you are also only checking whether max has includes on the top-level source, which isn't necessarily true for the max within this loop):

```suggestion
      if ( activity?.uses?.max?.includes?.(" ? ") ) activity.uses.max = "";
```

Alternately we can check if max is a string:

```suggestion
      if ( (foundry.utils.getType(activity?.uses?.max) === "string") && activity.uses.max.includes(" ? ") ) {
        activity.uses.max = "";
      }
```